monai/data/wsi_reader.py

```diff
@@ -322,6 +334,17 @@ def get_file_path(self, wsi) -> str:
         """Return the file path for the WSI object"""
         return self.reader.get_file_path(wsi)
 
+    def get_mpp(self, wsi, level: int) -> Tuple[float, float]:
```

Comment:
Maybe this `level` can be optional args, default is None then can use `self.level`.